### ISSUE TITLE
Add bottom bar backdrop

### ISSUE THREAD

**Issue by solargent** [robertodoering/harpy#434]
A nice-to-have I would love to see is the addition of some kind of (mostly opaque?) backdrop behind the bottom bar, because the secondary icons become difficult to see/read in certain scenarios (e.g. when there's text behind them). 

For example:

![image](https://user-images.githubusercontent.com/1126644/133345780-46df1d70-71c4-46dd-8c98-7e25d02bd48d.png)

**Comment by robertodoering** [comment]
I agree that it becomes a bit visually cluttered when the icons are above text. I think a nice in-between would be to keep the icons in 100% opacity for the non-selected tabs.
I will experiment a bit with it and see what works :) Thanks for the suggestion

**Comment by solargent** [comment]
I also wouldn't mind the option to just lock the bottom bar so it's always visible, but that's probably a separate thing.

**Comment by VitalTatun** [comment]
Also the same thing with status bar. Maybe will be better with no status bar at all, or with the theme color strip for example

**Comment by robertodoering** [comment]
@VitalTatun 
You can customize the color and transparency of the android status & navigation bar with custom themes (which will be available with the pro version)

**Comment by VitalTatun** [comment]
Now it looks like in the image attached,  and to get it normal you will have to get pro.... 🤷
![Uploading Screenshot_20210923-131542_Harpy Free.jpg…]()

**Comment by VitalTatun** [comment]
![Screenshot_20210923-131542_Harpy Free](https://user-images.githubusercontent.com/91216876/134491127-bc0f7b6d-cccd-4cee-a382-e129651b29b9.jpg)
